src/vouchedSession.js

```diff
@@ -43,6 +43,15 @@ export class VouchedSession {
             throw e
         }
     }
+
+    async postSelfieVerifaction(faceDetectionResult) {
```

Comment:
@JayLVouched I think it should be `postSelfieVerifacation`

Reply:
Good catch